### PR TITLE
Bugfix accordion (EXLM-378, 379)

### PR DESCRIPTION
**Pull request by gauthamksn**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: 

- https://jira.corp.adobe.com/browse/EXLM-378
- https://jira.corp.adobe.com/browse/EXLM-379

Test URLs:

Before: https://main--exlm--adobe-experience-league.hlx.page/docs/authoring-guide-exl/test/collapsible-content
After: https://bugfix-exlm-378-accordion--exlm--adobe-experience-league.hlx.page/docs/authoring-guide-exl/test/collapsible-content

### CODE REVIEW

File: blocks/accordion/accordion.scss [hunk 1]
```diff
@@ -14,9 +15,9 @@
       line-height: 21px;
 
       &::before {
-        display: inline-block;
         content: url('/icons/chevron_right.svg');
         position: relative;
+        margin-right: -5.6px;
```
Comment (ahmed-musallam) [inline blocks/accordion/accordion.scss:20]: Do we really need to do fractions?

File: blocks/accordion/accordion.scss [hunk 2]
```diff
@@ -91,4 +64,34 @@
...
+  .code code {
+    font-size: var(--spectrum-font-size-100);
+  }
+
+  .note-wrapper {
+    margin-left: 0;
+    margin-top: 0;
+  }
+
+  .note-wrapper div {
```
Comment (ahmed-musallam) [inline blocks/accordion/accordion.scss:87]: this is duplicated

File: blocks/accordion/accordion.scss [hunk 3]
```diff
@@ -91,4 +64,34 @@
...
+  .code-wrapper div {
+    margin-left: 0;
+    margin-top: 0;
+  }
+
+  .code code {
+    font-size: var(--spectrum-font-size-100);
+  }
+
+  .note-wrapper {
```
Comment (ahmed-musallam) [inline blocks/accordion/accordion.scss:82]: this is duplicated

File: blocks/accordion/accordion.css [hunk 4]
```diff
@@ -1,74 +1,76 @@
...
 }
-.accordion.block details summary::marker {
+.accordion details summary::marker {
   content: none;
 }
-.accordion.block details div > div {
+.accordion details div > div {
   color: var(--non-spectrum-raisin-black);
   font-size: var(--spectrum-font-size-200);
   line-height: 25.6px;
```
Comment (ahmed-musallam) [inline blocks/accordion/accordion.css:32]: do we really need to do fractions ?

**Review by ahmed-musallam — CHANGES_REQUESTED** [review]

**Review by thomsebastin — APPROVED** [review]

**Review by tprasad05 — APPROVED** [review]

**Review by ahmed-musallam — APPROVED** [review]

**Pull request merged**